### DISCUSSION TITLE
Ayuda con envío de trabajos

### DISCUSSION THREAD

**Discussion by MunozGalvanSandraLizbeth** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#148] [Q&A]
Hola, bueno lo que pasa es que desde ayer he estado intentado enviar mi repositorio de la misión FrontEnd en el formulario que nos dieron para subirlas, pero le doy en enviar y el botón no funciona... Dije a caray ¿ya cerró? Pero en la sección de arriba dice que se pueden enviar hasta el día 4 de marzo 
![image](https://user-images.githubusercontent.com/91628495/155856301-8421ce7f-8ac2-4cab-8883-d7108fa78868.png)

**Comment by Ingrzs** [comment]
Pues tendrás que esperar o mándale msj al commander en ig y el te dirá

**Comment by AngelCruzO** [comment]
Estas colocando todos los enlaces que te pide el formulario si es asi, intenta en otro navegador y que estes ingresando con tu correo de launch X

**Comment by avexyk** [comment]
El formulario funciona correctamente. Probablemente tengas problemas con algún campo que llenaste mal. En mi caso user Firefox. En Chrome y Edge debe funcionar correcto también.

**Comment by Burstniart** [comment]
Asegurate que estes en tu cuenta de **innovacción**, refesca el navegador y revisa que hayas llenado todos los campos.

**Comment by mgc14** [comment]
Hola, buen día,¿tienen el link para mandar lo de HTML? No encuentro el link por ningun lado.

Muchas gracias de antemano.

**Reply by davidtoString** [reply]
Lo de las practicas HTML de la pasteleria se enviarán en un enlace que se hará disponible el viernes 4 de marzo. Todavia no hay donde entregarlas.

**Comment by MunozGalvanSandraLizbeth** [comment]
Ya pude, gracias
